C/ChibiScheme/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+using BinaryBuilder
+
+name = "ChibiScheme"
+version = v"0.10"
+
+# Collection of sources required to build NLopt
+sources = [
+    GitSource("https://github.com/ashinn/chibi-scheme.git",
+              "05eb4ebd357e2bf2fe5aa7da13b0422ce20ddf7f"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/chibi-scheme
+
+apk add chibi-scheme
+
+make \
+    PREFIX="${prefix}" \
+    CC="${CC_BUILD}" \
+    AR="${AR_BUILD}" \
+    LD="${LD_BUILD}" \
+    CFLAGS="${CFLAGS}" \
+    CPPFLAGS="${CPPFLAGS}" \
+    LDFLAGS="${LDFLAGS}" \
+    RANLIB="${RANLIB_BUILD}" \
```

Comment:
Note that setting `X="${X_BUILD}"` has the effect of basically building for the host platform.  In fact audit of `x86_64-linux-gnu` (platform where our CI runs) fails to dlopen the library because of missing `libc.musl-x86_64.so.1` and audit on all platforms points out that the binary objects are for the wrong platform.

Side note: we don't set `CFLAGS`, `CPPFLAGS` or `LDFLAGS` (we force the relevant flags inside the compiler wrappers), so there is no need to forward them here, if that was the intent.

Reply:
Oh, I see.  (I think you mean that it builds for the *build* platform.  We would *like* to compile for the "host" platform in the usual cross-compilation parlance, if I understand correctly.)